tests/main.nf.test

```diff
@@ -0,0 +1,23 @@
+nextflow_pipeline {
+
+    name "Test pipeline"
+    script "../main.nf"
+    tag "pipeline"
+    tag "pipeline_differentialabundance"
+
+    test("Run with supplied profile") {
+
+        when {
+            params {
+                outdir     = "results"
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success }
```

Comment:
This is bare minimum. Added a snapshot with some of the results would be a great follow up!

Reply:
This is the pipeline level, i.e. there are no emits. What would be the way to snapshot?

Reply:
It is indeed the bare minimum, but I'm happy with that already to be honest